src/frontend/csrs-portal/src/app/app.module.ts

```diff
@@ -97,19 +98,18 @@ import { FormsModule } from '@angular/forms';
     CurrencyPipe,
     DatePipe,
     AppConfigService,
-    AuthConfigModule,
     {provide: MAT_DIALOG_DEFAULT_OPTIONS, useValue: {hasBackdrop: true}},
     {provide: STEPPER_GLOBAL_OPTIONS,useValue: {showError: true}},
     {
       provide: Configuration,
       useFactory: (authService: OidcSecurityService) => new Configuration(
         {
           basePath: '',//environment.apiUrl,
-          //accessToken: authService.tokengetAccessToken.bind(authService),
+          accessToken: authService.getAccessToken.bind(authService),
           credentials: {
             'Bearer': () => {
               var token: string = authService.getAccessToken();
-              if (token) {
+              if (token !== 'invalid_token') {
```

Comment:
Would we actually get anything like that from get Access token call? Or would we get that from token verification step? Thanks!

Reply:
This line 112 was corrected with previous text: " if (token) { "